odl/phantom/transmission.py

```diff
@@ -65,12 +71,14 @@ def _modified_shepp_logan_ellipses(ellipses):
     """
     intensities = [1.0, -0.8, -0.2, -0.2, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1]
 
+    assert len(ellipses) == len(intensities)
```

Comment:
I'm not so sure about that. To me, it depends on the nature of the internal helper. For instance, if the public function delegates everything, including checks, to internal helpers, I'm against asserts since they provide users with very little helpful information. They're not so internal in such cases, after all, and to me, this is such a case.

I'm okay with asserts when a deviation from the check is a bug rather than the result of invalid input. Internal consistency checks can use asserts, input validation should not in my opinion.